gammapy/irf/energy_dispersion.py

```diff
@@ -626,6 +626,48 @@ def offset(self):
         return self.data.axis('offset')
 
     @classmethod
+    def _from_gaus(cls, e_true, migra, bias, sigma, offset=None):
```

Comment:
:+1:  for the public method (like you put it). I can be useful e.g. for simulations